dalite_xblock/dalite_xblock.py

```diff
@@ -119,9 +131,7 @@ def launch_url(self):
         :returns: launch URL for selected Dalite-ng instance
         :rtype: string
         """
-        if not self.lti_passport:
-            return ''
-        return self.lti_passport.dalite_root_url.rstrip('/') + '/lti/'
+        return self._get_dalite_url('/lti/')
```

Comment:
`_get_dalite_url()` is only used once, right here.  So I don't think this change is necessary.